### DISCUSSION TITLE
zig + vscode 配置 debug 断点调试

### DISCUSSION THREAD

**Discussion by hhstore** [zigcc/forum#9] [General]
> 如题: 

- https://github.com/hhstore/blog/issues/375
- zig 断点调试, 含有完整示例代码 & 配置文件. 

> PS: zig + `vscode / sublime text / clion` 配置开发环境

- https://github.com/hhstore/blog/issues/376

**Comment by jiacai2050** [comment]
我一直都是 print debug 为主，偶尔会用用 lldb/gdb，不想花时间折腾所谓的 IDE 环境

**Reply by hhstore** [reply]
> 我一直都是 print debug 为主，偶尔会用用 lldb/gdb，不想花时间折腾所谓的 IDE 环境

嗯, 日常开发, 大多数场景 print + unit test 就够了. 
debug 断点, 适合跟踪 3rd lib/framework, 断点打在框架内, 方便分析/理解框架源码 & 定位/排查问题.
